packages/graphql-key-transformer/src/__tests__/KeyTransformer.test.ts

```diff
@@ -274,3 +274,96 @@ function getInputType(doc: DocumentNode, type: string): InputObjectTypeDefinitio
+  beforeEach(() => {
+    ff = {
+      getBoolean: jest.fn().mockImplementation((name, defaultValue) => {
+        if (name === 'skipOverrideMutationInputTypes') {
+          return true;
+        }
+      }),
+      getNumber: jest.fn(),
+      getObject: jest.fn(),
+      getString: jest.fn(),
+    };
+  });
+
+  it('@model mutation with user defined null args ', () => {
+    const validSchema = /* GraphQL */ `
+      type Call
+        @model(queries: null, mutations: null)
+        @key(fields: ["receiverId", "senderId"])
+        @key(name: "bySender", fields: ["senderId", "receiverId"]) {
+        senderId: ID!
+        receiverId: ID!
+      }
+
+      type Mutation {
+        createCall(input: CreateCallInput!): Call
+        deleteCall(input: DeleteCallInput!): Call
+      }
+
+      input CreateCallInput {
+        receiverId: ID!
+      }
+
+      input DeleteCallInput {
+        receiverId: ID!
+      }
+    `;
+    const transformer = new GraphQLTransform({
+      transformers: [new DynamoDBModelTransformer(), new KeyTransformer()],
+      featureFlags: ff,
+    });
+    const out = transformer.transform(validSchema);
+    expect(out).toBeDefined();
+    const schema = parse(out.schema);
+
+    const DeleteCallInput: InputObjectTypeDefinitionNode = schema.definitions.find(
+      d => d.kind === 'InputObjectTypeDefinition' && d.name.value === 'DeleteCallInput',
+    ) as InputObjectTypeDefinitionNode | undefined;
+    expect(DeleteCallInput).toBeDefined();
+    const receiverIdField = DeleteCallInput.fields.find(f => f.name.value === 'receiverId');
+    expect(receiverIdField).toBeDefined();
+    const senderIdField = DeleteCallInput.fields.find(f => f.name.value === 'senderId');
+    expect(senderIdField).toBeUndefined();
+  });
+
+  it('@model mutation with user defined create args ', () => {
+    const validSchema = /* GraphQL */ `
+      type Call
+        @model(queries: null, mutations: { delete: "testDelete" })
+        @key(fields: ["receiverId", "senderId"])
+        @key(name: "bySender", fields: ["senderId", "receiverId"]) {
+        senderId: ID!
+        receiverId: ID!
+      }
+
+      input CreateCallInput {
+        receiverId: ID!
+      }
+
+      input DeleteCallInput {
+        receiverId: ID!
+      }
+    `;
+    const transformer = new GraphQLTransform({
+      transformers: [new DynamoDBModelTransformer(), new KeyTransformer()],
+      featureFlags: ff,
+    });
+    const out = transformer.transform(validSchema);
+    expect(out).toBeDefined();
+    const schema = parse(out.schema);
+
+    const DeleteCallInput: InputObjectTypeDefinitionNode = schema.definitions.find(
+      d => d.kind === 'InputObjectTypeDefinition' && d.name.value === 'DeleteCallInput',
+    ) as InputObjectTypeDefinitionNode | undefined;
+    expect(DeleteCallInput).toBeDefined();
+    const receiverIdField = DeleteCallInput.fields.find(f => f.name.value === 'receiverId');
+    expect(receiverIdField).toBeDefined();
```

Comment:
Can you add assertion around type of the field and if its nullable or not for all the assertions.

Reply:
added assertions on fields